source-record.c

```diff
@@ -264,7 +265,10 @@ static void source_record_filter_offscreen_render(void *data, uint32_t cx,
 		return;
 
 	struct vec4 background;
-	vec4_zero(&background);
+	background.x = (double)(filter->backgroundColor & 0xff) / 255.0;
```

Comment:
why not use the vec4_from_rgba function?

Reply:
Fixed!